docs/editor_guide/segments_dashboard.rst

```diff
@@ -0,0 +1,84 @@
+The segments dashboard
+======================
+
+Wagxperience comes with two different views for it's segment dashboard. A "list
+view" and a "dashboard view". Where the dashboard view attempts to show all
+relevant information and statistics in a visually pleasing manner, the list view
+is more fitted for sites using large amounts of segments, as it may be
+considered more clear in these cases.
+
+
+Switching between views
+-----------------------
+
+By default, Wagxperience's "dashboard view" is active on the segment dashboard.
+If you'd like to switch between the dashboard view and list view, open the
+segment dashboard and click the "Switch view" button in the green header at the
+top of the page.
+
+.. figure:: ../_static/images/segment_dashboard_header.png
+   :alt: The header containing the "Switch view" button.
+
+
+Using the list view
+-------------------
+
+The advantages of using the list view:
+
+* Uses the familiar table view that is used on many other parts of the Wagtail
+  administration interface.
+* Offers a better overview for large amounts of segments.
+* Allows for reordering based on fields, such as name or status.
+
+.. figure:: ../_static/images/segment_list_view.png
+   :alt: The segment list view.
+
+
+Definitions
+^^^^^^^^^^^
+
+Name
+    The name of your segment.
+
+Persistent
```

Comment:
Je gaat hier weer uitleggen wat het allemaal doet, denk niet dat dat thuis hoort in de ``dashboard`` documentatie